.travis.yml

```diff
@@ -12,7 +12,10 @@ cache:
 
 before_script:
   - dartanalyzer .
-  - dartfmt --line-length=120 --dry-run --set-exit-if-changed .
+  - |
+    if [ "$TRAVIS_DART_VERSION" != "dev" ]; then
+      dartfmt --line-length=120 --dry-run --set-exit-if-changed .
```

Comment:
This fixes failures we were seeing in the Dart dev builds due to the dartfmt being different